### PR TITLE
Post dhall-1.20.1

### PR DESCRIPTION
**Pull request by jneira**
Mainly changes in dhall files format:
* remove `in let`
* replace old syntax for `Option`
* set default `dhall format` output for some files
* ...

### CODE REVIEW

File: dhall-to-cabal.dhall [hunk 1]
```diff
@@ -90,6 +90,7 @@ in    prelude.utils.GitHub-project
           "Distribution"
       -- build-type simple is needed to allow tools compatible with cabal < 2.2 build the package
       , build-type =
+          Some ( types.BuildType.Simple {=} )
           [ types.BuildType.Simple {=} ] : Optional types.BuildType
```
Comment (quasicomputational) [inline dhall-to-cabal.dhall]: This doesn't look right.

Comment (jneira) [reply dhall-to-cabal.dhall]: oops, bad merge, thanks for pointing out

Comment (jneira) [reply dhall-to-cabal.dhall]: Corrected, and tested with `dhall-to-cabal dhall-to-cabal.dhall`

**Comment by ocharles** [comment]
Thanks!

**Pull request merged**